201-encrypt-running-windows-vm-without-aad/README.md

```diff
@@ -0,0 +1,25 @@
+# Enable encryption on a running Windows VM. 
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F201-encrypt-running-windows-vm-without-aad%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F201-encrypt-running-windows-vm-without-aad%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/AzureGov.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-encrypt-running-windows-vm-without-aad%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template enables encryption on a running windows VM without needing AAD application. This template assumes that the VM is located in the same region as the resource group. If not, please edit the template to pass appropriate location for the VM sub-resources.

```

Comment:
doesn't this *have* to be true - i.e. the extension must be deployed in the same location as the VM (resourceGroup doesn't really matter) - if so, then we should provide a parameter for the vmLocation and just default it to [resourceGroup().location]

Reply:
Removed this statement to avoid confusion.